src/main/features/dashboard/views/macro/claimStatus/claimPartAdmittedDefendant.njk

```diff
@@ -22,22 +23,36 @@
 {% endmacro %}
 
 {% macro claimPartAdmittedToOweForDefendantClaimDetails(claim) %}
+  {% set claimant = claim.claimData.claimant %}
+  {% set defendant = claim.claimData.defendant %}
+  {% set claimantDetailsPageURI = DashboardPaths.contactThemPage.evaluateUri({ externalId: claim.externalId }) %}
+  {% set paymentIntention = claim.response.paymentIntention %}
+
   <div class="status-block">
     <h2 class="heading-small">{{ t('You’ve responded to the claim') }}</h2>
-    <p>{% if claim.response.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
+    <p>{% if paymentIntention.paymentIntention.paymentOption === domain.PaymentOption.IMMEDIATELY %}
```

Comment:
Doesn't look right... Should just be `paymentIntention.paymentOption` by the looks of it

Reply:
fixed